### ISSUE TITLE
background refresh is disabled

### ISSUE THREAD

**Issue by 532910** [home-assistant/iOS#2666]
**iOS device model, version and app version**
Model Name: iPhone 11
Software Version: 17.4.1
App version: 2024.2 (2024.561)

**Home Assistant Core Version**
2024.3.3

**Describe the bug**
`Background Refresh` is disabled though all permissions are provided.

**Screenshots**

**Comment by bgoncal** [comment]
Does rebooting the iPhone helps?

**Comment by 532910** [comment]
nope, I updated the ios for this report, so definitely no

**Comment by bgoncal** [comment]
Weird, in your settings (not from HA app) the "background refresh" toggle looks dimmed, check this out: https://discussions.apple.com/thread/255089291?sortBy=best

**Comment by 532910** [comment]
Indeed! It just was disabled globally. Is it possible to redirect from HA to the "background app refresh" screen? Anyway I believe HA could give some hints to check.

**Comment by bgoncal** [comment]
Thanks for the feedback, I'll look into it

**Comment by 532910** [comment]
Thank you, @bgoncal!
